midend/interpreter.cpp

```diff
@@ -676,16 +679,6 @@ void ExpressionEvaluator::postorder(const IR::ListExpression* expression) {
     set(expression, result);
 }
 
-void ExpressionEvaluator::postorder(const IR::StructExpression* expression) {
```

Comment:
why delete this function?

Reply:
It is very strange that I do not have this function in the file,I returned it